src/_includes/partials/global/site-foot.njk

```diff
@@ -10,7 +10,8 @@
         </div>
       {% endif %}
       <div>
-        {% include "partials/components/nav.njk" %}
+      {% set ariaLabel = 'footer navigation' %}
+      {% include "partials/components/nav.njk" %}
```

Comment:
I know it's super picky, but can we indent these in please?

Reply:
No worries at all! I guess prettier cannot format njk files. I struggle with a few other extensions also but they messed up the formatting completely. Do you just do the formatting manually?

Reply:
OK.. trying that again... some formatting went crazy. Let me know if this is ok now. Thanks for your help and patience!

Reply:
Most stuff gets caught by prettier, but I think nunjucks files get ignored. Thanks, I'll merge this in!